clearwater-cassandra/usr/share/clearwater/cassandra/cassandra-env.sh.template

```diff
@@ -137,6 +137,7 @@ esac
 
 #MAX_HEAP_SIZE="4G"
 #HEAP_NEWSIZE="800M"
+TEMPLATE_HEAPSIZE_SECTION
```

Comment:
Should we make it clear that these are sections that will be replaced in the final file?  As it is, they look like incomplete variable definitions.  For example, changing them to `<<<TEMPLATE_HEAPSIZE_SECTION>>>`?